### PR TITLE
Fixes #17723: Translate sles packer support in lua

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/17723

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

File: packer/scripts/cleanbox.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # Various cleanups to apply on the Vagrant boxes we use.
 # no set -e since some things are expected to fail
 
+set -x
+
```
Comment (peckpeck) [inline packer/scripts/cleanbox.sh:7]: Cleanbox is full of *** we should not have it

File: packer/systems/sles/scripts/os_server.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -xe
+
+# relay
+zypper --non-interactive install python3 openssl cron rsyslog apache apache2-utils postgresql
+# server
+zypper --non-interactive install iproute openldap2-client postgresql-server>=9.2 acl jre-headless\>=8
```
Comment (peckpeck) [inline packer/systems/sles/scripts/os_server.sh:8]: one > with backslash and one without ?

File: packer/systems/sles/sles.lua [hunk 3]
```diff
@@ -0,0 +1,20 @@
+function command(autoinst, other)
```
Comment (peckpeck) [inline packer/systems/sles/sles.lua:1]: There should be not "other" parameter in the sles version if you don't pass is when calling the function

**Comment by Fdall** [comment]
PR updated with a new commit

**Pull request merged**